api/share/requests/users.py

```diff
@@ -0,0 +1,65 @@
+import re
+from api.share.requests import ValidRequestObject, InvalidRequestObject
+
+
+class GetUsersCountRequest(ValidRequestObject):
+
+    def __init__(self,
+                 registration_bot_chat_id: str,
+                 tg_username: str,
+                 phone: str,
+                 email: str,
+                 first_name: str,
+                 last_name: str):
+        self.registration_bot_chat_id = registration_bot_chat_id
+        self.tg_username = tg_username
+        self.phone = phone
+        self.email = email
+        self.first_name = first_name
+        self.last_name = last_name
+
+    @classmethod
+    def from_dict(cls, input_dict):
+        # Check input parameters.
+        invalid_request = InvalidRequestObject()
+        # https://core.telegram.org/bots/api#chat : chat_id - max 52 significant bits
+        # 9007199254740991 is the max number you can store in the 53 bits => len(chat_id) <= 16
```

Comment:
```suggestion
        # 9007199254740991 is the max number you can store in the 53 bits. Hence len(chat_id) <= 16
```
It looks a bit confusing otherwise :)